explainer.md

```diff
@@ -181,6 +187,18 @@ const { doSomething } = await realm.import('./file.js');
 
 The Realms API does not introduce a new way to evaluate code, it is subject to the existing evaluation mechanisms such as the [Content-Security-Policy (CSP)](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy).
 
+## <a name='Whynotseparateprocesses'></a>Why not separate processes?
+
+Creating a Realm that runs in a separate process is another alternative, while allowing users to define and create their own protocol of communication between these processes.
+
+This alternative was discarded for two main reasons:
+
+1. There are existing mechanism to achieve this today in both browsers, and nodejs. E.g.: cross domain iframes, workers, etc. They seem to be good enough when asynchronous communication is sufficient to implement the feature.
+2. Asynchronous communication is a deal-breaker for many use-cases, and sometimes it just added complexity for cases where a same-process Realm is sufficient.
```

Comment:
we should add to this bullet that this is possible when security is NOT an issue, to avoid getting into this controversy around security. A good example here is AMP, they run in a cross domain iframe, they just want to have more control about what code they executed in that cross domain app.